CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ set(PROJECT_YEAR 2021)
 
 # needed for spec file
 execute_process (
-    COMMAND bash -c "uname -r | cut -d'.' -f4"
+    COMMAND bash -c "rpm -q glibc | cut -d'.' -f3"
```

Comment:
What's the rationale behind this? I used `uname` because it's available in our docker image by default where as rpm is not.

Reply:
But `uname -r` returns the release of the host. If you run a Photon 3.0 image in a Photon 4.0 host you get `4.0`. Even worse, when running on a Ubuntu host (as this workflow does), you get an empty string. That's how I noticed.

Reply:
Excellent catch. Yeah, this change makes sense. Thanks.

Reply:
I see though that this would break `cmake` when trying to build when `rpm` isn't installed. Maybe we should put the determination of the release tag into `ci/build-rpms.sh` and pass it to `rpmbuild` as a parameter.

Reply:
Or we can parse /etc/photon-release file

Reply:
`/etc/photon-release` would be Photon specific, but it should work for Fedora and other rpm based distros too. Also, it does not have the suffix. `/etc/os-release` does not work either, unless we would map `ID` and `VERSION`, but that would not be generic.

I am open to other suggestions. `glibc` is installed on most OSes. Should a use case come up that does not have it we can add whatever they use instead.